ayon_bin_distro/lakectlpy/wrapper.py

```diff
@@ -215,7 +286,13 @@ def clone_element(
                 sys.stdout.flush()
         return dest_path
 
-    def commit_local(self, commit_message: str, local_path=None):
+    def commit_local(self, commit_message: str, local_path: Optional[str] = None):
+        """commits changes from a local repository (like git commit -m)
+
+        Args:
+            local_path: optinal str needs to be a path if the current cwd is not the repository
+            commit_message: str commit message that will be send to lakeFs for the commit
```

Comment:
Arguments in docstring are in the wrong order, should match the function arguments.
```suggestion
        Args:
            commit_message (str): Commit message that will be send to lakeFs for the commit
            local_path (Optional[str]): Specify the local repository path, if not provided the
                current working directory (cwd) will be used.
```